contracts/LifCrowdsale.sol

```diff
@@ -69,6 +78,12 @@ contract LifCrowdsale is Ownable, Pausable {
     uint256 amount
   );
 
+  /**
+   * event for market market creation logging
+   * @param contractAddr the contract address
+   */
+  event MMCreated(address contractAddr);
```

Comment:
I'd not shorten the names here so I'd use `MarketMakerCreated` & `contractAddress`, what's the reason to shorten them?